src/eslint-config-adeira/runner/shouldLintAll.js

```diff
@@ -3,5 +3,5 @@
 export default function shouldLintAll(filename: string): boolean %checks {
   // Eslint configs can be nested (not only the root path).
   // See: https://eslint.org/docs/user-guide/configuring#configuration-file-formats
-  return /\.eslintrc(?:\.(?:js(?:on)?|ya?ml))?$|package.json$/.test(filename);
+  return /\.eslintrc(?:\.(?:js(?:on)?|ya?ml))?$|package.json$|.eslintignore$/.test(filename);
```

Comment:
I have this feeling that we should fix this regexp to ignore dots. Something like this:

```diff
-   return /\.eslintrc(?:\.(?:js(?:on)?|ya?ml))?$|package.json$|.eslintignore$/.test(filename);
+   return /\.eslintrc(?:\.(?:js(?:on)?|ya?ml))?$|package\.json$|\.eslintignore$/.test(filename);
//                                                        ^       ^
```

Reply:
You are right 🙈 Will do it right away

Reply:
#178